### PR TITLE
Implemented rudimentary pagination

### PR DESCRIPTION
**Pull request by gleono**
Here is the very simple pagination. I changed the service too, so it is easy to swap it for the javascript version later.

### CODE REVIEW

File: controllers/view.py [hunk 1]
```diff
@@ -33,20 +33,17 @@ def is_user_subscribed(self, stream_id):
...
         logout_url = users.create_logout_url('/')
 
         if not stream_id :
             self.display_error('The stream you are trying to access does not exist. It may have been removed by the owner.')
         else : 
-            view_api_uri = '{}?id={}'.format(self.uri_for('api-view', _full=True), stream_id)
-            stream_data = { 'id': stream_id }
-            json_data = json.dumps(stream_data);
-
+            view_api_uri = '{}?id={}&offset=0&limit={}'.format(self.uri_for('api-view', _full=True), stream_id, limit)
```
Comment (rayolanderos) [inline controllers/view.py:46]: Is the offset always 0? Meaning the pagination is more like a "view more"?

Comment (gleono) [reply controllers/view.py:46]: Yeah is a view more, that's what I see in the requirements. I left the offset so that JS later can take advantage of it.

Comment (rayolanderos) [reply controllers/view.py:46]: Gotcha.. I see the changes in the markup to reflect this behavior too.

**Review by rayolanderos — APPROVED** [review]
Looks great

**Review by gwoodard3 — APPROVED** [review]
I didn't see anything

**Pull request merged**